molpro/water_energy/input.py

```diff
@@ -0,0 +1,25 @@
+import qcengine as qcng
```

Comment:
Can you move this to a `molpro/scripts` or similar, but keep the input JSON example in the `water_energy/gradient` folders?

Reply:
So just to clarify, you mean move the input.py file to the `molpro/scripts` folder, but copy the JSON portion into a a file within the `water_energy/gradient` folder?

Reply:
Exactly!